CMakeLists.txt

```diff
@@ -112,9 +112,7 @@ include(autocmake_safeguards)
 find_package(PythonInterp REQUIRED)
 find_package(PythonLibs ${PYTHON_VERSION_MAJOR} REQUIRED)
 
-add_subdirectory(external)
-
-    # external projects manage their own OpenMP and c++YY flags, so only add to CXX_FLAGS for psi4_core
+# external projects manage their own OpenMP and c++YY flags, so only add to CXX_FLAGS for psi4_core
```

Comment:
I go back and forth on what's best for including external projects in a super build.  My current thought is it's best to forward all flags to the underlying projects so everything in the build uses the same options.  You only avoid passing certain options to them if those options interfere with their code.